.github/workflows/job_PreCommit_PythonAutoformatter.yml

```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Python Precommit Formatter
+on:
+  pull_request_target:
+    branches: [ "master", "release-*" ]
+    paths: [ "model/**","sdks/python/**","release/**"]
+  issue_comment:
+    types: [created]
+  push:
```

Comment:
The `push` event should also include a list of `paths`. Please apply this to all the job files below.

Reply:
Not sure of the logic here. I went with all paths as to have complete code tested once in master. Ill change it but its up to check what is the real wanted behaviour

Reply:
Done. If different ill change it again